int32  robot_id
float64 rssi
